corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -131,6 +131,104 @@
 
     {% if not use_js_bundler %}
       {% javascript_libraries use_bootstrap5=use_bootstrap5 underscore=True jquery_ui=request.use_jquery_ui ko=True hq=True analytics=True %}
```

Comment:
What do you think of adding a comment to this block to the effect of
```html
{# Everything in this `if not use_js_bundler`-block below this line can be removed once all of app manager, including form builder, is on webpack. #}
```

It doesn't seem obvious, and seems like it could be easily forgotten.

Reply:
I will not forget 🙂  I've got a note to get rid of `use_js_bundler` altogether. But adding a comment sounds fine - 0969128dd8b